src/Response/ProtocolVersion.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Jasny\HttpMessage\Response;
+
+use Jasny\HttpMessage\Message;
+
+/**
+ * ServerRequest protocol version methods
+ */
+trait ProtocolVersion
+{
+    use Message\ProtocolVersion;
+
+    /**
+     * Determine the protocol versions based on the server params
+     * 
+     * @return string
+     */
+    protected function determineProtocolVersion()
+    {
+        return "1.1";
```

Comment:
Should be persistent. If response defaults to 1.1 than ServerRequest should do the same.